core/jvm/src/test/scala/com/dwolla/consul/examples/ConsulMiddlewareAppPlatform.scala

```diff
@@ -23,6 +23,10 @@ trait ConsulMiddlewareAppPlatform extends IOApp.Simple {
   override def run: IO[Unit] =
     jaegerEntryPoint[IO]
       .flatMap(_.root("ConsulMiddlewareApp"))
-      .evalMap(new ConsulMiddlewareApp[Kleisli[IO, Span[IO], *]].run.run)
+      .evalMap {
+        implicit val loggerFactory = Slf4jFactory.create[IO]
```

Comment:
It seems kind of silly to me to have to do this explicitly, but the implicit was deprecated, so it's here to keep the compiler warnings clean. 🤷‍♂️

There's more discussion in issue 683 over at typelevel/log4cats, for future code archeologists.